src/main/java/seedu/command/student/AddStudent.java

```diff
@@ -2,36 +2,33 @@
 
 import seedu.StudentList;
 import seedu.command.Command;
-import seedu.duke.Duke;
 import seedu.exception.DukeException;
+import seedu.ui.DisplayList;
 import seedu.ui.UI;
 
-import java.util.ArrayList;
-
 import static seedu.duke.Duke.studentListCollection;
 
 public class AddStudent extends Command {
 
-    static UI ui = new UI();
+    private static UI ui = new UI();
```

Comment:
I think no need to declare as static, because the only use of ui (line25) doesn't access any static function/variable.

Reply:
Also try to instantiate (i.e. `new`) in constructor instead of here.